prisma/schema.prisma

```diff
@@ -92,3 +93,13 @@ model Invite {
   createdBy   User @relation(fields: [createdById], references: [id])
   createdById Int
 }
+
+model Limit {
+  id         Int      @id @default(autoincrement())
+  type_time  String   @default("daily")
+  limit_by   String   @default("count")
```

Comment:
I think using an enum for `type_time` and `limit_by` would be better than using String.